### DISCUSSION TITLE
Should we keep the changelog up-to-date or can we remove it?

### DISCUSSION THREAD

**Discussion by lewisblake** [metno/pyaerocom#1183] [Polls]
We have a changelog directory where we store a markdown files of the changlogs. It's a little redundant because all of this info comes from git log anyway. It also can mess up searches when looking for keywords because many hits occur in these files. If no one is using them or finds them helpful, I would consider removing them.

**Comment by heikoklein** [comment]
I like a well curated changelog, but the data in the changelog-directory are just an automated git-excerpt of PR-header or commit-logs without any added value.

**Reply by lewisblake** [reply]
Agreed about a well curated change log in principle. We also have the change logs from the GitHub releases page, which is quite useful. https://github.com/metno/pyaerocom/releases
